flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/CallExpression.java

```diff
@@ -216,6 +217,17 @@ public String asSummaryString() {
         return getFunctionName() + argList;
     }
 
+    @Override
+    public String asSerializableString() {
+        if (functionDefinition instanceof BuiltInFunctionDefinition) {
+            return ((BuiltInFunctionDefinition) functionDefinition)
+                    .getCallSyntax()
+                    .unparse(((BuiltInFunctionDefinition) functionDefinition).getSqlName(), args);
+        } else {
+            return CallSyntax.FUNCTION.unparse(getFunctionName(), args);
```

Comment:
We should throw an exception when `functionIdentifier` is null. Something like "Only functions that have been registered before are serializable"?